Meme/Meme/Sources/Wiki/Presentation/ViewModel.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  ViewModel.swift
+//  Meme
+//
+//  Created by 제나 on 7/20/25.
+//
+
+import Combine
+
+// FIXME: - 파일없으면 깃에서 안보여서 있는 임시 파일
+
+extension Wiki.ViewModel {
+    final class Main: Wiki.Namespace {
+        private let lobbyUseCase: Wiki.UseCase.LobbyUseCase
```

Comment:
```suggestion
        private let lobbyUseCase: UseCase.LobbyUseCase
```

이거는 단순 샘플 코드라 그런거 같은데 나중에 작업들어가면 이런식으로 통일하는게 좋아보인다.